### PR TITLE
Enikshay fix and debug view

### PR DESCRIPTION
**Pull request by sravfeyn**
@proteusvacuum

### CODE REVIEW

File: custom/enikshay/tasks.py [hunk 1]
```diff
@@ -179,6 +182,24 @@ def is_dose_taken(cases):
...
+        # return property 'adherence_schedule_date_start' of episode case, cast to datetime
+        raw_date = self.get_property('adherence_schedule_date_start')
+        if not raw_date:
+            return None
+        if parse_datetime(raw_date):
+            return parse_datetime(raw_date)
+        elif parse_date(raw_date):
+            return datetime.datetime.combine(parse_date(raw_date), datetime.datetime.min.time())
+        else:
+            logger.error(
```
Comment (proteusvacuum) [inline custom/enikshay/tasks.py]: this is the only place where the logger is being used, is that intentional?

File: custom/enikshay/tasks.py [hunk 2]
```diff
@@ -179,6 +182,24 @@ def is_dose_taken(cases):
 
         return dose_taken_by_date
 
+    def get_adherence_schedule_start_date(self):
+        # return property 'adherence_schedule_date_start' of episode case, cast to datetime
+        raw_date = self.get_property('adherence_schedule_date_start')
```
Comment (proteusvacuum) [inline custom/enikshay/tasks.py]: I know you didn't change this here, but I think every call to `dynamic_case_properties` does a DB lookup. You might want to cache that in some way (e.g. `self.properties`) in the `get_property` method.

**Review by proteusvacuum — APPROVED** [review]
couple of questions, otherwise g2g

File: custom/enikshay/tasks.py (hunk 1 above)
Comment (sravfeyn) [inline custom/enikshay/tasks.py]: Good catch, it was mistake. I meant to raise an exception here, updated now

**Pull request merged**